src/main.js

```diff
@@ -423,7 +423,6 @@ function configureWindowEvents() {
     });
 
     ipcMain.handle("translations", async () => {
-      translations = await i18next.getDataByLanguage(userData.data.language).translation
```

Comment:
This is was duplicate on my side. As the translations are already loaded earlier (line 631) we only need to return those here, so they are being passed back on to the renderer process.